mmv1/products/dialogflowcx/SecuritySettings.yaml

```diff
@@ -0,0 +1,161 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'SecuritySettings'
+base_url: 'projects/{{project}}/locations/{{location}}/securitySettings'
+update_verb: :PATCH
+update_mask: true
+description: |
+  Represents the settings related to security issues, such as data redaction and data retention. It may take hours for updates on the settings to propagate to all the related components and take effect.
+  Multiple security settings can be configured in each location. Each agent can specify the security settings to apply, and each setting can be applied to multiple agents in the same project and location.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/dialogflow/cx/docs'
+  api: 'https://cloud.google.com/dialogflow/cx/docs/reference/rest/v3/projects.locations.securitySettings'
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 40
+  update_minutes: 40
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'dialogflowcx_security_settings_basic'
+    primary_resource_id: 'basic_security_settings'
+    vars:
+      settings_name: 'dialogflowcx-security-settings'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'dialogflowcx_security_settings_full'
+    primary_resource_id: 'basic_security_settings'
+    vars:
+      inspect_name: 'dialogflowcx-inspect-template'
+      deidentify_name: 'dialogflowcx-deidentify-template'
+      settings_name: 'dialogflowcx-security-settings'
+      bucket_name: 'dialogflowcx-bucket'
+    test_env_vars:
+      project: :PROJECT_NAME
+id_format: 'projects/{{project}}/locations/{{location}}/securitySettings/{{name}}'
+import_format: ['projects/{{project}}/locations/{{location}}/securitySettings/{{name}}']
+skip_sweeper: true
```

Comment:
I think we can generate a sweeper here- did you run into any issues?

Reply:
Nope! The code I copy-pasted from (`Agent.yaml`) had `skip_sweeper: true` and I see that all the other resource files under dialogflowcx do as well; let's see if tests pass!